### PR TITLE
add note on readme

### PR DESCRIPTION
**Pull request by tigercosmos**
Add note about how to use command while running servo
origin  `readme` is not clear.  
I think ./mach run [url] [arguments] is more clear than ./mach run [url]

---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [ ] There are tests for these changes OR
- [ ] These changes do not require tests because _____

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/19628)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -243,6 +243,13 @@ URL with servo).
 
 ### Commandline Arguments
 
+Usage:
+
+```sh
+./servo [url] [command] # if you run with nightly build
```
Comment (emilio) [inline README.md]: nit: maybe `[arguments]`? instead of `[command]`?

File: README.md [hunk 2]
```diff
@@ -243,6 +243,13 @@ URL with servo).
...
+./mach run [url] [command] # if you run with mach
```
Comment (emilio) [inline README.md]: Ditto.

**Review by emilio — COMMENTED** [review]
We already have a `Running` section right above this section, should we just update that?

File: README.md (hunk 1 above)
Comment (tigercosmos) [reply README.md]: sure. I will change

**Comment by tigercosmos** [comment]
@emilio I change to `Running` section.
I think `./mach run [url] [argument]` is more clear than `./mach run [url]`

File: README.md [hunk 3]
```diff
@@ -230,9 +230,13 @@ cp servobuild.example .servobuild
 
 ## Running
 
-Use `./mach run [url]` to run Servo. Like so,
+Run Servo with the command:
 
-``` shell
+```sh
+./servo [url] [argument] # if you run with nightly build
+./mach run [url] [argument] # if you run with mach
```
Comment (CYBAI) [inline README.md]: Will using plural be much more clear?

Comment (tigercosmos) [reply README.md]: no matter I thought

Comment (KiChjang) [reply README.md]: `arguments` makes more sense to me.

**Comment by KiChjang** [comment]
@bors-servo r+

Thanks!

**Pull request merged**